server/controllers/article.js

```diff
@@ -41,3 +39,22 @@ export const getArticles = async (req, res) => {
     displayError(err, res, 500);
   }
 };
+
+export const getAnArticle = async (req, res) => {
+  try {
+    const { articleId } = req.params;
+
+    const article = await Article.findOne({
+      where: { id: articleId }
+    });
+
+    if (article === null) {
+      handleFailedResponse(null, 'No article with this id found', res, 404);
```

Comment:
I don't think we need `handleFailedResponse`. I would suggest you use `displayError();`